instrumentation/mysql/src/test/java/brave/mysql/ITTracingStatementInterceptor.java

```diff
@@ -34,6 +34,7 @@
 
 public class ITTracingStatementInterceptor {
   static final String QUERY = "select 'hello world'";
```

Comment:
in a unit test I think you want the old condition. I'm worried that Math.min won't work because if space is present and \n is not, won't it return -1 instead of the index of space?